app/scripts/directives/pupilteacherratiochart.coffee

```diff
@@ -0,0 +1,39 @@
+'use strict'
+
+###*
+ # @ngdoc directive
+ # @name edudashApp.directive:ratePassChart
+ # @description
+ # # ratePassChart
+###
+angular.module 'edudashAppDir'
+.directive 'pupilTeacherRatioChart', [
+    '$log'
+    ($log) ->
+      restrict: 'E'
+      templateUrl: 'views/pupilteacherratiochart.html'
```

Comment:
If we add
```coffeescript
    scope: selectedSchool: '=selectedSchool'
```
to the directive, then we can pass `selectedSchool` as an `attr` and make the dependency explicit, and make this directive a little safer.